core/src/main/groovy/com/novoda/gradle/release/ReleasePlugin.groovy

```diff
@@ -43,8 +42,12 @@ class ReleasePlugin implements Plugin<Project> {
         }
     }
 
-    private String getString(Project project, String propertyName, String defaultValue) {
-        project.hasProperty(propertyName) ? project.getProperty(propertyName) : defaultValue
+    private String getProjectProperty(Project project, String propertyName, String defaultValue) {
+        if (project.hasProperty(propertyName) && project.getProperty(propertyName) != 'unspecified') {
```

Comment:
seems that `unspecified` is a default, would be nice to pull this check out into a `isDefault(propertyName)`